splight_lib/models/metadata.py

```diff
@@ -17,7 +17,7 @@ class Metadata(SplightDatabaseBaseModel):
     @model_validator(mode="after")
     def check_value_type(cls, model):
         model.value = (
-            None
+            model.value
```

Comment:
Let me know if you think this is going to break something

Reply:
Why is this change?